tests/unit/coordinator_instance.cpp

```diff
@@ -45,71 +45,92 @@ class CoordinatorInstanceTest : public ::testing::Test {
   std::vector<uint32_t> const coordinator_ids = {41, 42, 43};
   std::vector<uint16_t> const bolt_ports = {2687, 2688, 2689};
   std::vector<uint16_t> const coordinator_ports = {40113, 40114, 40115};
+  std::vector<uint16_t> const management_ports = {20113, 20114, 20115};
 };
 
 // Empty until you run 1st RegisterReplicationInstance or AddCoordinatorInstance
 TEST_F(CoordinatorInstanceTest, ShowInstancesEmptyTest) {
-  auto const init_config =
-      CoordinatorInstanceInitConfig{coordinator_ids[0], coordinator_ports[0], bolt_ports[0],
-                                    main_data_directory / "high_availability" / "coordinator", "localhost"};
+  auto const init_config = CoordinatorInstanceInitConfig{coordinator_ids[0],
+                                                         coordinator_ports[0],
+                                                         bolt_ports[0],
+                                                         management_ports[0],
+                                                         main_data_directory / "high_availability" / "coordinator",
+                                                         "localhost"};
 
-  auto const instance1 = CoordinatorInstance{init_config};
-  auto const instances = instance1.ShowInstances();
+  auto instance1 = CoordinatorInstance{init_config};
```

Comment:
CONST

Reply:
Can't be const because ShowInstances is not const. ShowInstances creates connections, so it can't be const